docs/applications/emby.md

```diff
@@ -0,0 +1,107 @@
+# Emby
+
+Homepage: [https://emby.media/](https://emby.media/)
+
+Emby is a mostly open-source media server with a client-server model. This
+install for Ansible-NAS provides a server, which various clients can then
+connect to from various platforms - other computers, smartphones, smart TVs,
+etc.
+
+Note that [Plex](https://www.plex.tv/), also included in Ansible-NAS, has a very
+similar functionality.
+
+## Usage
+
+Set `emby_enabled: true` in your `group_vars/all.yml` file. There are further
+parameters you can edit such as `movies_root` and `tv_root` lower down in this
+file. 
+
+## Specific Configuration
+
+The emby web interface can be found at port 8096 (http) or 8920 (https, if
+configured) of your NAS. After installation, point your browser there to start
+the configuration wizard. For instance, on a NAS with the IPv4 address
+192.168.1.2, browse to `http://192.168.1.2:8096`. Heimdall has a dedicated icon
+for emby. 
+
+By default, Ansible-NAS gives emby read/write access to the folders where your
+movies and TV shows are stored. To change this to read-only, edit the following
+lines in `tasks/emby.yml`
+
+```
+      - "{{ emby_movies_directory }}:/movies:rw"
+      - "{{ emby_tv_directory}}:/tv:rw"
+```
+
+so that they end in `ro` instead of `rw`. Note that emby will not be able to
+delete files then, which might be exactly what you want. However, you will not
+have the option to store cover art in the related folders. Always leave the
+configuration directory read/write. 
+
+## File system considerations
```

Comment:
This is great info, but not specific to Emby. Perhaps could be moved out to a ZFS page?

Reply:
You're right, we should probably have a "What you need to know about ZFS for Ansible NAS" page somewhere, including a setup for snapshots. I'll open an issue for that and move stuff over there.

(Is it just me or has interest in ZFS on Linux exploded since Ubuntu got behind it? I'm also seeing  a lot more stuff about Proxmox suddenly, but that might just be the company I keep)